website/integrations: rename Hoarder to Karakeep

petrsimunek opened the pull request:
<!--
👋 Hi there! Welcome.

Please check the Contributing guidelines: https://docs.goauthentik.io/docs/developer-docs/#how-can-i-contribute
-->

## Details

<!--
Explain what this PR changes, what the rationale behind the change is, if any new requirements are introduced or any breaking changes caused by this PR.

Ideally also link an Issue for context that this PR will close using `closes #`
-->
The Hoarder app has been renamed to Karakeep recently. https://www.reddit.com/r/selfhosted/comments/1js667o/hoarder_is_rebranding_to_karakeep/

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ X] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

petrsimunek committed:
Hoarder renamed to karakeep

The Hoarder app has been renamed to Karakeep recently. https://www.reddit.com/r/selfhosted/comments/1js667o/hoarder_is_rebranding_to_karakeep/

Signed-off-by: petrsimunek <49954958+petrsimunek@users.noreply.github.com>

dominic-r commented:
(The file itself also needs to be renamed ^^)

tanberry commented:
> (The file itself also needs to be renamed ^^)

True, that would be ideal for many reasons (prevent future confusion, SEO) but if we chnage file name to `karakeep.md` we will need to add a redirect from old file path to new. I will try to remember to do so when we merge this. Thanks @petrsimunek for catching this name change!

petrsimunek commented:
Hi all :) , ok, how should I proceed now? :) Should I create a new pull request with the changes? (filename and sidebar)

tanberry commented:
> Hi all :) , ok, how should I proceed now? :) Should I create a new pull request with the changes? (filename and sidebar)

Sorry, I was unclear. You won;t need to submit a new PR, you can just push those two chnages to this PR. Yes, please add Karakeep to the `sidebar.js` file and rename `hoarder.md` file to be `karakeep.md`. (Good catch @dominic-r .)

Let us know if any questions.

petrsimunek committed:
folder hoarder renamed to karakeep

petrsimunek committed:
sidebar changed from hoarder to karakeep

petrsimunek commented:
Hi @tanberry , many thanks for clarification. :-) I am a newbie in colaboration on github, still learning.
I renamed the folder with `index.md` containing information about the karakeep/hoarder (it is based on folder name, not file name apparently) and switched hoarder for karakeep in `sidebarIntegrations.js`

tanberry commented:
Thanks again @petrsimunek for this PR!
